ci/jenkins/pipelines/prs/skuba-jjb-validation.Jenkinsfile

```diff
@@ -51,7 +51,7 @@ pipeline {
         } } }
 
         stage('Setting GitHub in-progress status') { steps {
-            sh(script: "${PR_MANAGER} update-pr-status ${GIT_COMMIT} ${PR_CONTEXT} 'pending'", label: "Sending pending status")
+            sh(script: "${PR_MANAGER} update-pr-status ${PR_CONTEXT} 'pending'", label: "Sending pending status")
         } }
 
         stage('Git Clone') { steps {
```

Comment:
Probably remove?

Reply:
Not sure what you are asking here. Remove what?

Reply:
The stage that reclones the repo under the subdirectory that this PR removes.